### PR TITLE
I18N: glossary: New Term "Dashboard"

### PR DESCRIPTION
**Pull request by CWehli**

### CODE REVIEW

File: po/glossary/gnc-glossary.txt [hunk 1]
```diff
@@ -76,6 +76,7 @@
 "credit note"	"A document that you give to a client that says you owe money to the client, i.e. the opposite of an invoice"
 "currency"	"The system of money used in a country"
 "Custom"	"Custom print format (i.e. according to the user's wishes) as opposed to a template choice."
+"Dashboard"	"-"
```
Comment (fellen) [inline po/glossary/gnc-glossary.txt:79]: Without an  explanation it does not make much sense.

**Review by fellen — CHANGES_REQUESTED** [review]

File: po/glossary/gnc-glossary.txt (hunk 1 above)
Comment (CWehli) [reply po/glossary/gnc-glossary.txt]: @fellen: Done.

Comment (fellen) [reply po/glossary/gnc-glossary.txt]: I should have searched it before: https://en.wikipedia.org/wiki/Dashboard_(disambiguation)
Is everything important included?

Comment (CWehli) [reply po/glossary/gnc-glossary.txt]: @fellen: Is my suggestion so bad?

Comment (fellen) [reply po/glossary/gnc-glossary.txt]: I don't know it. I didn't know the word and have not used the report. So I posted just an idea and will trust you.

**Review by fellen — COMMENTED** [review]
Ich habe noch den passenden wp link angehangen.

**Pull request merged**